src/mixins/canvas_dataurl_exporter.mixin.js

```diff
@@ -85,20 +85,21 @@ fabric.util.object.extend(fabric.StaticCanvas.prototype, /** @lends fabric.Stati
 
     var croppedCanvasEl,
         croppedCtx,
+        croppedNodeCanvasEl,
         shouldCrop = 'left' in cropping ||
                      'top' in cropping ||
                      'width' in cropping ||
                      'height' in cropping;
 
     if (shouldCrop) {
-
       croppedCanvasEl = fabric.util.createCanvasElement();
+      croppedNodeCanvasEl = fabric.isLikelyNode ? canvasEl._nodeCanvas : canvasEl;
```

Comment:
this looks like a big change.
or am i wrong? looks like that canvasEl is no more the same between browser and nodecanvas